moveit_ros/move_group/src/default_capabilities/apply_planning_scene_service_capability.cpp

```diff
@@ -39,29 +39,41 @@
 
 namespace move_group
 {
+
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_move_group_default_capabilities.apply_planning_scene_service_capability");
```

Comment:
I see, it's really super long. I would probably just remove the "default_capabilities" in the middle, actually.